src/components/Delete.jsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteHouseThunk } from '../Redux/deleteSlice.js';
+import { Link, useNavigate } from 'react-router-dom';
+import { userHouses } from '../Redux/HouseSlice.js';
+
+const DeleteHouse = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+  dispatch(userHouses());
+  },[dispatch]);
+
+  const userhouses = useSelector((state) => state.Houses.userhouses);
+  const navigate = useNavigate();
+
+  const handleDelete = (houseId) => {
+    dispatch(deleteHouseThunk(houseId)).then((action) => {
```

Comment:
add this suggetions to update the user houses list.
```suggestion
await dispatch(deleteHouseThunk(houseId)).then((action) => {
    dispatch(userHouses())
```

Reply:
It is done. can I have little description sis

Reply:
yeah sure. This is for after deletion it needs to get the newest list of user houses or else it will give an error.